network/network_stats.ml

```diff
@@ -119,7 +119,9 @@ let read_stats () =
 			if payload |> Digest.string |> Digest.to_hex <> checksum then
 				raise Invalid_checksum
 			else
-				payload |> Jsonrpc.of_string |> stats_t_of_rpc
```

Comment:
Why do we need to change this instead of keeping the `deriving rpc` annotation and using the generated `stats_t_of_rpc`?

Reply:
By `deriving rpcty`, there are no `stats_t_of_rpc` again.

Reply:
Why do we need to derive `rpcty` instead of `rpc`? Can we do `[@@deriving rpc, rpcty]`?

Reply:
John requests using rpcty, I think derive both from `rpc` and `rpcty` works, but it's redundant because they handle the same stuffs.

Reply:
In this case I don't see the benefit. @jonludlam why `rpcty` for this instead of `rpc`?